walletconsole/lib/Address.cpp

```diff
@@ -80,16 +80,16 @@ bool Address::addrDefault(const string& coinid, string& res) {
 }
 
 bool Address::addrDP(const string& coinid, const string& derivPath, string& res) {
```

Comment:
`addrDP` what is the meaning of this function?

Reply:
it means: get address from derivation path (part of wallet console)